api/coverage.py

```diff
@@ -68,25 +70,43 @@ def __init__(self, _db, identifier_types=None, metadata_lookup=None,
                 Identifier.GUTENBERG_ID, 
                 Identifier.AXIS_360_ID,
             ]
-        self.coverage_source = DataSource.lookup(
+        self.output_source = DataSource.lookup(
             self._db, DataSource.METADATA_WRANGLER
         )
 
         if not metadata_lookup:
-            url = Configuration.integration_url(
-                Configuration.METADATA_WRANGLER_INTEGRATION
-            )
-            metadata_lookup = SimplifiedOPDSLookup(url)
+            metadata_lookup = SimplifiedOPDSLookup.from_config()
         self.lookup = metadata_lookup
 
         super(MetadataWranglerCoverageProvider, self).__init__(
             self.service_name,
             identifier_types,
-            self.coverage_source,
+            self.output_source,
             workset_size=20,
             cutoff_time=cutoff_time,
         )
 
+    @property
+    def items_that_need_coverage(self):
+        reaper_source = DataSource.lookup(
+            self._db, DataSource.METADATA_WRANGLER_COLLECTION
```

Comment:
Rather than create special data sources for the metadata wrangler collection and the reaper, how about using the 'operation' field of CoverageRecord? Then you can use METADATA_WRANGLER as the data source instead of creating a new data source.